matRad/doseCalc/+DoseEngines/matRad_ParticleAnalyticalBortfeldEngine.m

```diff
@@ -324,6 +326,11 @@ function calcLateralParticleCutOff(this,cutOffLevel,~)
                 calcRange = true;
             end
 
+            if strcmp(this.cutOffMethod, 'relative')
+                warning('Relative cutoff not yet implemented for Analytical Engine. Default integral method used.')
+                this.cutOffMethod = 'integral';
+            end
+
             for i = 1:numel(this.machine.data)
                 this.machine.data(i).LatCutOff.CompFac = 1/cutOffLevel;
                 this.machine.data(i).LatCutOff.numSig  = sqrt(2) * sqrt(gammaincinv(0.995,1)); %For a 2D symmetric gaussian we need the inverse of the incomplete Gamma function for defining the CutOff
```

Comment:
Actually I also think we should simplify this. No need to use `gammaincinv` here, as the radial  Gaussian is simple enough to use 
`sqrt(-2*log(0.005))` instead of `sqrt(2) * sqrt(gammaincinv(0.995,1))`.
However, we should fix that we use the hardcoded `0.995` and not the `cutOffLevel`.
So: `sqrt(-2*log(1-cutOffLevel))`